src/app/planner/containers/planner.component.ts

```diff
@@ -56,4 +73,51 @@ export class PlannerComponent implements OnInit {
                 });
             }).unsubscribe();
     }
+
+    public eventModify(event) {
+        if (!this.confirmationDialogOpen) {
```

Comment:
shouldn't we revert FC-event action when this dialog is open?

Reply:
There can be only one dialog of such kind so it's not necessary. Also, clicking away from the dialog (i.e anywhere in the background) defaults to reverting the changes.